packages/playwright-test/src/plugins/vitePlugin.ts

```diff
@@ -113,6 +113,10 @@ export function createPlugin(
       if (hasNewTests || hasNewComponents || sourcesDirty)
         await fs.promises.writeFile(buildInfoFile, JSON.stringify(buildInfo, undefined, 2));
       previewServer = await preview(viteConfig);
+      const isAddressInfo = (x: any): x is AddressInfo => x?.address;
+      const address = previewServer.httpServer.address();
+      if (isAddressInfo(address))
```

Comment:
And if it's not AddressInfo, we silently do nothing? Under what circumstances do we not get an address from .address()?

Reply:
That would be an imaginary use case.

Reply:
> Under what circumstances do we not get an address from .address()?

So none! It should always return an address—and the code assumes that—so why the fancy `is` type discriminator, instead of just a cast? Stylistic or am I missing something?